api/services/itemService.js

```diff
@@ -0,0 +1,15 @@
+const { itemDao } = require("../models");
+
+const teabags = async (data, category) => {
+    const seeTeabags = await itemDao.seeTeabags(data, category);
+    return seeTeabags;
```

Comment:
```suggestion
    return itemDao.seeTeabags(data, category);
```
이미 충분히 자명하게 변수명이 작성되어 있어 별도의 변수 선언 없이 곧바로 리턴해도 깔끔할 것 같습니다~